bridges/snowbridge/pallets/ethereum-client/src/mock.rs

```diff
@@ -144,6 +112,13 @@ impl ethereum_beacon_client::Config for Test {
 pub fn new_tester() -> sp_io::TestExternalities {
 	let t = frame_system::GenesisConfig::<Test>::default().build_storage().unwrap();
 	let mut ext = sp_io::TestExternalities::new(t);
-	let _ = ext.execute_with(|| Timestamp::set(RuntimeOrigin::signed(1), 30_000));
+	let _ = ext.execute_with(|| {
+		let inbound_fixture = snowbridge_pallet_ethereum_client_fixtures::make_inbound_fixture();
+		EthereumBeaconClient::store_finalized_header(
+			inbound_fixture.finalized_header,
+			inbound_fixture.block_roots_root,
+		)
+		.unwrap();
+	});
```

Comment:
Should all beacon unit tests do this? Seems unnecessary, let's just import it for the relevant test?

Reply:
https://github.com/Snowfork/polkadot-sdk/pull/125/commits/16482488d48c6e80bfde8f3a607315ca72625562